src/utils/file/index.ts

```diff
@@ -185,3 +187,12 @@ export async function convertForPreview(file: File, fileBlob: Blob) {
     }
     return fileBlob;
 }
+
+export function isSharedFile(file: File) {
+    const user: User = getData(LS_KEYS.USER);
+
+    if (!user.id || !file.ownerID) {
+        return false;
+    }
```

Comment:
Curious about the need for such defensive check. Ideally, they should never be nil right if isSharedFile is getting called?

Reply:
ya..ideally it should never happen... just wanted to avoid a false positive case (`undefined!==undefined` is `true`)